tests/debug-tests/utils.py

```diff
@@ -0,0 +1,87 @@
+import argparse
+
+from distributed.comm.utils import from_frames
+from distributed.utils import nbytes, parse_bytes
+
+import numpy as np
+import ucp
+
+ITERATIONS = 50
```

Comment:
Should this also be `from utils import ITERATIONS`?

Reply:
same as above